src/consensus/event_accumulator.rs

```diff
@@ -303,11 +303,7 @@ mod test {
     }
 
     fn empty_elders_info() -> EldersInfo {
-        unwrap!(EldersInfo::new_for_test(
-            Default::default(),
-            Default::default(),
-            Default::default(),
-        ))
+        EldersInfo::new(Default::default(), Default::default(), Default::default())
```

Comment:
Yeah, maybe. On the other hand, `EldersInfo` with all default values isn't useful and is only used here because for the purpose of these tests it's sufficient. So I'd probably keep it as it, to at least highlight the fact that something non-standard is going on here.

Reply:
ok, got it